zapcore/encoder_test.go

```diff
@@ -21,10 +21,13 @@
 package zapcore_test
 
 import (
+	"encoding/json"
 	"strings"
 	"testing"
 	"time"
 
+	"gopkg.in/yaml.v2"
+
```

Comment:
nit: delete this blank line between third party imports

Reply:
✅ c6de7e9c783b51707bc7962284be5f08d86883f6